lk2nd/device/dts/msm8909/apq8009w-wtp.dts

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+        model = "Qualcomm APQ8009W";
+        qcom,msm-id = <QCOM_ID_APQ8009 0>, <QCOM_ID_APQ8009W 0>;
+        qcom,board-id = <QCOM_BOARD_ID_MTP 0x105>;
+};
+
+&lk2nd {
+        mobvoi-catfish {
+                model = "Mobvoi TicWatch Pro (WF12096)";
+                compatible = "mobvoi,catfish";
+                lk2nd,dtb-files = "mobvoi,catfish";
+                lk2nd,match-panel;
+                panel {
+                        compatible = "mobvoi,catfish-panel";
+                        qcom,mdss_dsi_rm67162_wqvga_cmd {
+                                compatible = "mobvoi,rm67162";
+                        };
+                };
+                gpio-keys {
+                        compatible = "gpio-keys";
+                        volume-down {
+                                gpios = <&tlmm 98 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+                                lk2nd,code = <KEY_VOLUMEDOWN>;
+                        };
+                };
+        };
+};
```

Comment:
```suggestion
// SPDX-License-Identifier: GPL-2.0-only

#include <skeleton64.dtsi>
#include <lk2nd.dtsi>

/ {
	model = "Qualcomm APQ8009W";
	qcom,msm-id = <QCOM_ID_APQ8009 0>, <QCOM_ID_APQ8009W 0>;
	qcom,board-id = <QCOM_BOARD_ID_MTP 0x105>;
};

&lk2nd {
	mobvoi-catfish {
		model = "Mobvoi TicWatch Pro (WF12096)";
		compatible = "mobvoi,catfish";
		lk2nd,dtb-files = "mobvoi,catfish";
		lk2nd,match-panel;

		panel {
			compatible = "mobvoi,catfish-panel";

			qcom,mdss_dsi_rm67162_wqvga_cmd {
				compatible = "mobvoi,rm67162";
			};
		};

		gpio-keys {
			compatible = "gpio-keys";

			volume-down {
				gpios = <&tlmm 98 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
				lk2nd,code = <KEY_VOLUMEDOWN>;
			};
		};
	};
};
```

Minor coding style fixes:
- Use tabs instead of spaces
- Newline between properties and nodes.

Reply:
Thanks! (The spaces instead of tabs was a mistake when I copied it from another PC)